templates/cluster-template.yaml

```diff
@@ -579,9 +579,8 @@ spec:
       cluster:
         name: ${NUTANIX_PRISM_ELEMENT_CLUSTER_NAME}
         type: name
-      image:
-        name: ${NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME}
-        type: name
+      imageLookupBaseOS: ${BASE_OS}
+      imageLookupFormat: ${LOOKUP_FORMAT}
```

Comment:
this probably shouldn't be in the base template - it causes the e2e tests fail. id like some guidance on how to deal with this.

Reply:
Did you define BASE_OS and LOOKUP_FORMAT somewhere for E2E tests?

Reply:
i will start a new e2e test for this feature.